client/src/index.module.scss

```diff
@@ -1,11 +0,0 @@
-body {
-  margin: 0;
```

Comment:
Please, add margin: 0 on body to global styles

Reply:
Done to App.module.scss as index.module.scss is not a needed file.